client/packages/coldchain/src/Monitoring/ListView/TemperatureChart/TemperatureTooltipLayout.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTheme } from '@common/styles';
+import { Box, Typography, useFormatDateTime } from '@openmsupply-client/common';
+import React from 'react';
+
+type Entry = {
```

Comment:
it's a temperature log entry rather than a sensor. You could argue that the colour is attached to a sensor - but the tooltip is when you over over an entry on the chart, so I think Entry is still an ok name.